### PR TITLE
feat: Add @chydeepak7 as a contributor

### LINKED ISSUE #337: Feature:Add @GITHUB-USERNAME as a contributor
### Description

As a part of the Intro to Open Source, we are encouraged to contribute to this repo.

### Suggested solution

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by chydeepak7**
## Description
This PR add @chydeepak7 as a contributor

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [x] 📝 Documentation Update

## Related Issues
Fixes #337 

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)
image

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @chydeepak7,

Thank you for your contribution! ✨ 

There are a few things here:

- There are failed checks that you can [read here](https://github.com/open-sauced/guestbook/pull/339#pullrequestreview-2081237187). Please read the bot messages and fix the problems until all check pass.
- You opened two PRs. Please close one that you don't use because they are duplication.
- **Very Important:**  You [last commit](https://github.com/open-sauced/guestbook/pull/339/commits/76250a08d648d6c14aa8d5f985cd8bb5cf25b572) deleted the `README.md` and modify the `package-lock.json`. This action will break the production. Please revert the commit. You can read [this blog post](https://dev.to/github/how-to-undo-pushed-commits-with-git-2pe6) to do so.

**Comment by chydeepak7** [comment]
Hello @adiati98 ,
I have deleted the duplicated PR and also reverted the last commit. Can you please check if I have done something wrong.
And as for the bot message I am not able to understand what its saying can you give me a clue of what I should do?

**Comment by chydeepak7** [comment]
package-lock.json how do i revert it I dont know in which commit i changed it

**Comment by adiati98** [comment]
@chydeepak7, look at the last point in my previous comment. I link your commit there.

**Comment by adiati98** [comment]
As for the only bot message left, it's saying that you modified the `package-lock.json`, and a maintainer should carefully review it.

In this case, we (maintainers) don't want to modify this file at all. That's why you need to revert the file. 🙂

**Comment by chydeepak7** [comment]
I think all the checks have passed now.

File: README.md [hunk 1]
```diff
@@ -36,7 +42,11 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## All Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<<<<<<< HEAD
 [![All Contributors](https://img.shields.io/badge/all_contributors-141-orange.svg?style=flat-square)](#contributors-)
+=======
+[![All Contributors](https://img.shields.io/badge/all_contributors-142-orange.svg?style=flat-square)](#contributors-)
+>>>>>>> parent of 76250a0 (some minor changes)
```
Comment (adiati98) [inline README.md:49]: There's a duplication for the badge here. You need to remove line 47-49.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @chydeepak7,

When you resolve conflicts, you need to remove the `<<<`, `===`, and `>>>`.

After you address all the reviews, don't forget run the command to generate your profile again in your terminal because now the states are change.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
+ma
+=======
+=======
+>>>>>>> parent of 76250a0 (some minor changes)
```
[the remark this answers is not in the document]
Comment (chydeepak7) [reply README.md:6]: I checked my README.md but there are no such changes as shown above.

[the remark this answers is not in the document]
Comment (chydeepak7) [reply README.md:6]: I tried in github codespace and those lines are showing. But in my vs code those lines are absent. Why is that??

**Comment by adiati98** [comment]
Hey @chydeepak7 
You haven't run the generate command. 

Please address all the suggestions before requesting for review. 🙂

**Comment by chydeepak7** [comment]
> Hey @chydeepak7 You haven't run the generate command.
> 
> Please address all the suggestions before requesting for review. 🙂

I just did it right now.

**Comment by adiati98** [comment]
@chydeepak7 I see that the badge is not increasing.
Have you run the `npm run contributors:generate` command in your terminal?

**Comment by chydeepak7** [comment]
> @chydeepak7 I see that the badge is missing instead of increasing. Have you run the `npm run contributors:generate` command in your terminal?

yes

**Review by adiati98 — APPROVED** [review]
Well done, @chydeepak7! ✨ 

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P).

git merge GIF

**Comment by chydeepak7** [comment]
Thank you for your time and guidance

**Pull request merged**